### PR TITLE
Brython Proposal

### PR DESCRIPTION
**Pull request by NadieFiind**

### CODE REVIEW

File: bin/pynani.py [hunk 1]
```diff
@@ -161,21 +144,26 @@ def build(self):
...
-		except:
-			pynani.create_app("My Nani App")
-	elif sys.argv[1] == "runserver":
-		try:
-			pynani.run_server(sys.argv[2])
-		except:
-			pynani.runserver()
-except:
-	pynani.pynani_help()
+    print(sys.argv[1])
```
Comment (jabezborja) [inline bin/pynani.py:148]: You can remove this, this was just for debugging.

File: bin/pynani.py [hunk 2]
```diff
@@ -245,3 +233,20 @@ def server_start(port):
...
+            name = "My App"
+        
+        try:
+            description = sys.argv[3]
+        except IndexError:
+            description = "My PyNani application."
+        
+        create_app(name, description)
+except IndexError:
+    pynani_help()
```
Comment (jabezborja) [inline bin/pynani.py:252]: You can insert if on:

```
if __name__ == '__main__':...
```

**Review by jabezborja — COMMENTED** [review]
Looks good.

File: bin/pynani.py (hunk 2 above)
Comment (NadieFiind) [reply bin/pynani.py:252]: I also added the description as the 4th argument so you could create app now by using `pynani.py create-app MyApp "This is the description."`.

File: README.md [hunk 3]
```diff
@@ -81,12 +81,12 @@ source ~/.bashrc
...
 To create a Web App, go to the folder where you want PyNani to install in then go to CMD or command line and type:
 ```
-pynani.py create-app <App_Name>
+pynani.py create-app <App_Name> <App_Description>
 ```
 
 in real world example, it should be:
 ```
-pynani.py create-app my_cool_app
+pynani.py create-app my_cool_app "Cool app description."
```
Comment (jabezborja) [inline README.md:89]: Oh, what is the matter with inputting the App name and App description?

Comment (NadieFiind) [reply README.md:89]: Nothing. But I think that it is more neat like that.
I can revert it back if you don't want it that way.

Comment (jabezborja) [reply README.md:89]: No, it's good. Don't revert it.

File: bin/pynani.py [hunk 4]
```diff
@@ -107,36 +98,15 @@ def build(self):
...
-    }
-</style>
-</html>
-    """)
-
-    settings.writelines(f"""app_name: {APP_NAME}
-description: {DESCRIPTION}
-    """)
+    with open(os.path.abspath(path(os.path.dirname(__file__), "..", "core", "public", "index.html"))) as file:
+        indexHTML.writelines(file.read())
```
Comment (jabezborja) [inline bin/pynani.py:105]: COOL!

File: bin/pynani.py [hunk 5]
```diff
@@ -205,33 +175,35 @@ def on_modified(self, event):
...
         observer.stop()
+    
     observer.join()
 
-def server_start(port):
-    PORT = port if port == "" else 8000
-
-    web_dir = os.path.abspath(path(os.path.dirname( __file__ ), '..', 'core', "public"))
+def server_start(port: str):
+    PORT = int(port) if port else 8000
```
Comment (jabezborja) [inline bin/pynani.py:191]: I was also planning to fix this because it is a string.

File: bin/pynani.py [hunk 6]
```diff
@@ -205,33 +175,35 @@ def on_modified(self, event):
...
     os.chdir(web_dir)
 
     handler = http.server.SimpleHTTPRequestHandler
 
     with socketserver.TCPServer(("", PORT), handler) as httpd:
         print(f"Server now started at localhost: {PORT}")
-        print("PyNani version that you're currently using is 1.0.0-alpha")
-        print("To quit the server or to turn off exit the command-line")
+        print(f"PyNani version that you're currently using is {VERSION}")
+        print("To stop the server, press CTRL+C or exit the command-line")
```
Comment (jabezborja) [inline bin/pynani.py:201]: CTRL+C to kill the server will never work. The Thread is stopping it from working. It is forever (of course if you close the file)

**Review by jabezborja — COMMENTED** [review]
So far, so good! Thanks for cleaning my shitty codebase. Good job! LGTM (if you're finished)

**Pull request closed without merging**